x-pack/plugin/transform/src/main/java/org/elasticsearch/xpack/transform/checkpoint/TimeBasedCheckpointProvider.java

```diff
@@ -95,7 +95,7 @@ public void createNextCheckpoint(final TransformCheckpoint lastCheckpoint, final
         // for time based synchronization
         final long timeUpperBound = alignTimestamp.apply(timestamp - timeSyncConfig.getDelay().millis());
 
-        getIndexCheckpoints(ActionListener.wrap(checkpointsByIndex -> {
+        getIndexCheckpoints(null, ActionListener.wrap(checkpointsByIndex -> {
```

Comment:
As above, would it be better than what we have today to set a huge timeout like 12 hours here?